test/tsconfig.json

```diff
@@ -13,6 +13,7 @@
   "include": [
     "**/*.ts",
     "**/*.tsx",
+    "../typings/lodash.topath/*.ts",
```

Comment:
do we need this import? why not add the whole `typings` folder ?

Reply:
Yeah we do need this because we're relying on some internal import from lodash. I couldn't import all of the `typings` directory because some of the files depends on the `jest` global being defined.